repositories.yaml

```diff
@@ -2028,3 +2028,25 @@ applications:
     channels:
       - v1_name: glam
         app_id: glam
+
+  - app_name: subscription_platform_backend
+    canonical_app_name: Subscription Platform
+    app_description: |
+      Mozilla Subscription Platform is Mozilla's subscription management
+      solution for account-based end-user services and features.
+    url: https://github.com/mozilla/fxa
+    notification_emails:
+      - dalvarez@mozilla.com
+    branch: main
+    metrics_files:
+      - libs/shared/metrics/glean/src/registry/subplat-backend-metrics.yaml
+    ping_files: []
+    dependencies:
```

Comment:
They're not sure in the bug. I added glean-server because @Dexterp37 mentioned that it seems like it. But how can I tell?
I also added glean-js because the bug mentions glean on NPM.

Reply:
@edugfilho I don't think it can be both glean-server and glean-js. I was assuming backend because I though subplat was backend, but might be wise to ask the requester.